geode-lucene/src/main/java/org/apache/geode/cache/lucene/internal/LuceneIndexForPartitionedRegion.java

```diff
@@ -171,15 +176,32 @@ private PartitionResolver getPartitionResolver(PartitionAttributes dataRegionAtt
     configureLuceneRegionAttributesFactory(partitionAttributesFactory, partitionAttributes);
 
     // Create AttributesFactory based on input RegionShortcut
+    final boolean isOverFlowToDisk = isOverFlowToDisk(regionAttributes);
     RegionAttributes baseAttributes = this.cache.getRegionAttributes(regionShortCut.toString());
     AttributesFactory factory = new AttributesFactory(baseAttributes);
     factory.setPartitionAttributes(partitionAttributesFactory.create());
     if (regionAttributes.getDataPolicy().withPersistence()) {
       factory.setDiskStoreName(regionAttributes.getDiskStoreName());
     }
+    if (isOverFlowToDisk) {
+      factory.setEvictionAttributes(regionAttributes.getEvictionAttributes());
```

Comment:
what does using overflow to disk have to do with per region tombstone sweepers? This should be a seperate checkin